README.md

```diff
@@ -1,6 +1,32 @@
-ooc-kean
+magic-sdk
 ========
+**Last updated: 2016-01-18**
 
-ooc port of parts of Kean
+This is an extended SDK for `magic`, a high-level cross-platform language which transpiles into C99.
 
-Uses [Sean Barrett's stb library](https://github.com/nothings/stb) for opening and saving images.
+## History
+This project was originally a fork of the [ooc project](https://github.com/fasterthanlime/rock) and contains ports of selected parts of the C# library [Kean](https://github.com/cogneco/Kean). The most notable changes include dropping garbage collection. It also uses [Sean Barrett's stb library](https://github.com/nothings/stb) for opening and saving images.
+
+## How to run
+To compile, you need the Cogneco version of the [Rock compiler](https://github.com/cogneco/rock). To make life easy, you might also want an editor with ooc/magic syntax highlighting support and a style enforcer. Those are available (for Ubuntu) at the [magic-tools repo](https://github.com/cogneco/magic-tools#installation-ubuntu), which can also be used to automatically install (and update) rock.
+
```

Comment:
https://github.com/cogneco/magic-tools#installation-ubuntu should now be https://github.com/magic-lang/magic-tools#installation-ubuntu